### PR TITLE
Prettify JSON and YAML VIB files

### PR DESCRIPTION
**Pull request by carrodher**
### Description of the change

All JSON and YAML files present in the `.vib` directory were prettified by running
```
$ yq r --tojson YAML_FILE | jq . | yq r --prettyPrint - > YAML_FILE.new && mv YAML_FILE.new YAML_FILE
$ jq . JSON_FILE > JSON_FILE.new && mv JSON_FILE.new JSON_FILE
```

### Benefits

All files are formatted and follow the same conventions.

### CODE REVIEW

File: .vib/airflow/cypress/cypress.json [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-    "env": {
-      "username": "user",
-      "password": "ComplicatedPassword123!4"
-    }
+  "baseUrl": "http://localhost",
+  "pageLoadTimeout": 240000,
+  "defaultCommandTimeout": 80000,
+  "env": {
+    "username": "user",
+    "password": "ComplicatedPassword123!4"
```
Comment (marcosbc) [inline .vib/airflow/cypress/cypress.json:7]: It seems like this is not sorted

Comment (marcosbc) [reply .vib/airflow/cypress/cypress.json:7]: Using `jq --sort-keys .` should solve this.

Comment (carrodher) [reply .vib/airflow/cypress/cypress.json]: Done

File: .vib/apache/vib-verify.json [hunk 2]
```diff
@@ -1,51 +1,53 @@
...
           "action_id": "helm-package"
         },
         {
           "action_id": "helm-lint"
         }
-      ]
-    },
-    "verify": {
+      ],
       "context": {
```
Comment (superaleks) [inline .vib/apache/vib-verify.json:12]: In the previous version, there was a `verify` before the `context`. 
Now, I don't see it. Could it affect the functionality of the pipeline?

Comment (carrodher) [reply .vib/apache/vib-verify.json:12]: It shouldn't be removed but ordered, from
```
{
    "phases": {
      "package": {
        "context": ...,
        "actions": ...
      },
      "verify": {
        "context": ...,
        "actions": ...
      }
    }
  }
```
to
```
{
  "phases": {
    "package": {
      "actions": ...,
      "context": ...
    },
    "verify": {
      "actions": ...,
      "context": ...
    }
  }
}
```

**Comment by recena** [comment]
@carrodher Could you update this PR?

**Comment by carrodher** [comment]
@recena I just updated the branch by rebasing master, it was a pain due to the number of changes. It would be nice if @superaleks @joancafom @FraPazGal can take a look to avoid more conflicts in the future.

In the same way, we should do something (GH Action) to apply this kind of linter/prettify for new PRs

**Comment by recena** [comment]
@carrodher We should add some linter into the PR-verify pipeline

**Comment by carrodher** [comment]
I will close this PR and address it with a proper internal task

**Pull request closed without merging**